src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -586,9 +585,7 @@ public Throwable getLastError() {
 				return connectTask.thenCompose((webSocket) -> {
 					return this.sendAsyncLock.acquireThenCompose(duration, () -> {
 						CloseReason reason = new CloseReason(CloseCodes.NORMAL_CLOSURE, "Normal Closure");					
-						return webSocket.closeAsync(reason).whenComplete(($void, ex) -> {
-							this.onOffline(ex);
-						});
```

Comment:
Offline handler was called twice. Removing this one here because the other one is located after the ReceivePump exits, which is executed even when the close is unexpected.